### PR TITLE
N&S edits for Persisting Data

### PR DESCRIPTION
**Pull request by devinmarieb**
# Description
- Update the location of the page in the side nav
- Add note about not being able to customize duration of storage
- Change bold note of billable storage to blue info note
- Put link to page inside Caching page

# Reasons
[Jira Ticket](https://circleci.atlassian.net/jira/software/projects/DOCSTEAM/boards/412?selectedIssue=DOCSTEAM-114) - first ticket on the N&S agenda
The page needed some minor updates and the location of the page in the nav has been changed for greater visibility.

### CODE REVIEW

**Comment by rosieyohannan** [comment]
Looks great! Maybe add a link to the persisting data page from https://circleci.com/docs/2.0/artifacts/ too?

**Comment by rosieyohannan** [comment]
Might be good to create a whole "persisting data" section/subsection with the Persisting data page as the overview/intro page, then including the caching and artifacts docs, and the workspaces doc once it's been pulled out of the workflows page. Let's keep the change you have here - moving persisting data under pipelines for now though, and we can reassess in a separate ticket.

File: jekyll/_cci2/caching.md [hunk 1]
```diff
@@ -46,7 +46,7 @@ Caching keys are simple to configure. The following example updates a cache if i
 {: #introduction }
 {:.no_toc}
 
-Automatic dependency caching is not available in CircleCI, so it is important to plan and implement your caching strategy to get the best performance. Manual configuration in 2.0 enables more advanced strategies and finer control.
+Automatic dependency caching is not available in CircleCI, so it is important to plan and implement your caching strategy to get the best performance. Manual configuration in 2.0 enables more advanced strategies and finer control. See the [Persisting Data]({{site.baseurl}}/2.0/persist-data/) page for  tips on caching strategies.
```
Comment (thomasr8) [inline jekyll/_cci2/caching.md:49]:
```suggestion
Automatic dependency caching is not available in CircleCI, so it is important to plan and implement your caching strategy to get the best performance. Manual configuration in 2.0 enables more advanced strategies and finer control. See the [Persisting Data]({{site.baseurl}}/2.0/persist-data/) page for tips on caching strategies.
```

**Review by thomasr8 — APPROVED** [review]
This is great @devinmarieb ! Just found one minor extra space.

**Pull request merged**